include/kernels/prolongator.cuh

```diff
@@ -0,0 +1,111 @@
+#include <color_spinor_field_order.h>
+#include <multigrid_helper.cuh>
+#include <kernel.h>
+
+namespace quda {
+
+  using namespace quda::colorspinor;
+  
+  static constexpr int fine_colors_per_thread = 1; // for all grids use 1 color per thread
```

Comment:
I'd prefer this wasn't defined in the "global" namespace (ignoring being in the `quda` namespace). I understand this needs to be accessed by multiple structs/classes, can this be made a public static member of `ProlongateArg` that other classes (`Prolongator`, `Prolongate` [in the appropriate *.cu]) can query? Or put in a `prolongate` namespace?

I'm receptive to other ideas, too, I'd just like it out of a relatively public space.

Reply:
Or it could be made a `template` argument again, like it is in `develop`.

Reply:
Or a `constexpr` function like what's in `restrictor.cuh`, even that feels cleaner...

Reply:
I've changed this to the restrictor style as suggested (44877dd73)